### PR TITLE
Implement resource manager and gRPC services

### PR DESCRIPTION
**Pull request by Jeffwan**
## Why are these changes needed?
Implement the gPRC service and provide the interface for cluster CRUD operations.

## Related issue number
#87 

## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [] Unit tests
   - [x] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: apiserver/pkg/manager/resource_manager.go [hunk 1]
```diff
@@ -10,3 +41,242 @@ func NewResourceManager(clientManager ClientManagerInterface) *ResourceManager {
...
+
+	return newRuntime, nil
+}
+
+func (r *ResourceManager) GetComputeTemplate(ctx context.Context, name string) (*v1.ConfigMap, error) {
+	client := r.clientManager.KubernetesClient().ConfigMapClient(DefaultNamespace)
+	return getComputeTemplateByName(ctx, client, name)
+}
+
+func (r *ResourceManager) ListComputeTemplates(ctx context.Context) ([]*v1.ConfigMap, error) {
```
Comment (chenk008) [inline apiserver/pkg/manager/resource_manager.go:183]: Is computeTemplate saved in configmap?

**Comment by chenk008** [comment]
LGTM, thanks!

**Review by chenk008 — APPROVED** [review]

File: apiserver/pkg/manager/resource_manager.go (hunk 1 above)
Comment (Jeffwan) [reply apiserver/pkg/manager/resource_manager.go:183]: Yes. That's true. We can change to db or annotations if needed later.

**Pull request merged**